test/cases/compile/require-context/index.js

```diff
@@ -0,0 +1,6 @@
+it("should maintain require context", function() {
+	var context = {foo: "bar"};
+	require(["./a", "./b"], function(a, b) {
+		this.foo.should.eql("bar");
+	}.bind(context));
```

Comment:
@sokra, updated test case per review comments.  However, as previously indicated, this still doesn't catch the error being fixed because depBlock.chunks.length is never greater than 1 in lib/depdendencies/DepBlockHelpers when when this code is compiled by the test harness.  A chunk optimizer plugin seems to be needed in order to make that happen.